Yoast/javascript
# Lin 269 test words ending in t or d

## Summary

This PR can be summarized in the following changelog entry:

* [yoastseo] Add final check for words ending in t/d for cases that are not covered by already existing checks. This check is meant to reduce the cases of incorrectly stemmed words ending in t/d.

## Relevant technical choices:

* Exception list of (the most frequent) words ending in t/d is compiled (up to 5000 words).
* This list is further checked before we decide to stem further the word ending in t/d.

## Test instructions

This PR can be tested by following these steps:

* Add more words ending in t/d in the spec (either in stemTOrDFromEndOfWordSpec or in determineStemSpec.
* run yarn test and make sure that there is no failing test.

## Quality assurance


Fixes #
